Translation corrections (#350)
07th-mod/higurashi-console-arcs: Update/omot_003.txt

//r――そんな期待を完全に捨て去れない、自分の弱さに気づくたびに、余計自己嫌悪の感情が募ってゆく。…まさに、悪循環だった。
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "——そんな期待を完全に捨て去れない、自分の弱さに気づくたびに、余計自己嫌悪の感情が募ってゆく。…まさに、悪循環だった。",

> 			NULL, "...Whenever I noticed how weak I was in that I refused to completely disregard that sort of explanation, it added to my self-loathing. ...It was a vicious cycle.", Line_Normal);

Reviewer: ```suggestion
			NULL, "...Whenever I noticed how weak I was for refusing to completely disregard that sort of explanation, it added to my self-loathing. ...It was a vicious cycle.", Line_Normal);
```

			NULL, " It seems a love triangle has formed around Satoko!", Line_WaitForInput);
	ModPlayVoiceLS(4, 6, "ps3/s14/06/210600156", 256, TRUE);
	OutputLine(NULL, "　なんか私たちのバックに百合の花が見えますよ～？」",

> 			NULL, " Is that yuri I see sprouting up in the background between us all~?\"", Line_Normal);

Reviewer: ```suggestion
			NULL, " Is that yuri I see sprouting up in the background between all of us~?\"", Line_Normal);
```

	if (GetLocalFlag(LConsoleArc)==10) {
		ModPlayVoiceLS(4, 4, "ps2/04/210400100", 540, TRUE);
		OutputLine(NULL, "「わ、私たちの部活って、こんなに皆さんから認知されていたのでございますのね……！」",

> 				NULL, "\"I-I had no idea every single one of them were aware of our club activities...!\"", Line_Normal);

Reviewer: ```suggestion
				NULL, "\"I-I had no idea every single one of them was aware of our club activities...!\"", Line_Normal);
```

	} else {
		ModPlayVoiceLS(4, 4, "ps3/s14/04/vt4ba_satoko1010", 256, TRUE);
		OutputLine(NULL, "「わ、私たちの部活って、こんなに皆さんから認知されておりましたのね…！」",

> 				NULL, "\"I-I had no idea every single one of them were aware of our club activities...!\"", Line_Normal);

Reviewer: ```suggestion
				NULL, "\"I-I had no idea every single one of them was aware of our club activities...!\"", Line_Normal);
```
